static/js/components/ShareDataForm.jsx

```diff
@@ -173,6 +173,16 @@ const ShareDataForm = ({ route }) => {
             {route.id}
           </Link>
         </Typography>
+        <p>
+          This page allows you to share data for {`${route.id}`} with other
+          users or groups. Select the photometry or spectra you would like to
+          share from the list below, then select the users or groups you would
+          like to share the data with. When you click submit, the access
+          permissions on the data will be updated. This page does <em>not</em>
+          reproduce the &ldquo;transfer source&ldquo; functionality of the
+          GROWTH marshal. Data shared via this page will not cause the source to
```

Comment:
We would like to keep SkyPortal survey-agnostic, so mentioning the GROWTH marshal should not happen here.

Reply:
Ah, good point @stefanv ! @dannygoldstein could you please remove that sentence?